pupil-spa/src/environments/environment.ts

```diff
@@ -6,6 +6,9 @@
 export const environment = {
   apiURL: 'http://localhost:3001',
   production: false,
-  apiErrorDelay: 2000,
-  apiErrorMaxAttempts: 3
+  checkStartAPIErrorDelay: 2000,
+  checkStartAPIErrorMaxAttempts: 3,
+  checkSubmissionApiErrorDelay: 30000,
```

Comment:
It would be nice (for me) if the units were specified, or if the config file was commented.

Reply:
@jon-shipley on it

Reply:
@jon-shipley updated